besu/src/test/java/org/hyperledger/besu/PrivacyReorgTest.java

```diff
@@ -92,9 +93,9 @@
       Bytes.fromBase64String("A1aVtMxLCUHmBVHXoZzzBgPbW/wj5axDpW9X8l91SGo=");
 
   private static final String FIRST_BLOCK_WITH_NO_TRANSACTIONS_STATE_ROOT =
-      "0x1bdf13f6d14c7322d6e695498aab258949e55574bef7eac366eb777f43d7dd2b";
+      "0xc9dcaffbebc7edc20839c80e706430a8fa885e8f703bb89f6e95633cc2b05d4d";
   private static final String FIRST_BLOCK_WITH_SINGLE_TRANSACTION_STATE_ROOT =
-      "0x16979b290f429e06d86a43584c7d8689d4292ade9a602e5c78e2867c6ebd904e";
+      "0x66fbc6ad12ef1da78740093ea2b3362e773e510e27d8f88b68c27bfc1f4d58c8";
```

Comment:
@lucassaldanha this test broke because I changed the BlockDataGenerator. I think it's safe to change the state roots here but I don't think the test should rely on hardcoded state roots anyways. Is there a way to determine at test time what the correct state roots should be?

Reply:
I think actually this is not a bad thing to rely on hardcoded state roots on tests. Because when you change something it breaks, and this is good. If you use your code to generate the assertion there is a risk of working only in close loop.